css/style.css

```diff
@@ -0,0 +1,169 @@
+html, body {
+    background: #ECEEF1;
+}
+
+.container {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: center;
+  }
+#account {
```

Comment:
[Optioal] Not required to complete this project.  Consider using **classes**  instead of i**ds** classes are reusable ids are not.

Reply:
Hello Ispirett, the suggested changes have been fixed and pushed.